model/src/w3strkmd.F90

```diff
@@ -315,7 +315,7 @@ SUBROUTINE waveTracking_NWS_V2 (intype     ,tmax       , &
     !/       reserved.  WAVEWATCH III is a trademark of the NWS.
     !/       No unauthorized use without permission.
     !/
-    USE W3SERVMD, ONLY: EXTIOF
+    USE W3SERVMD, ONLY: EXTIOF, EXTCDE
```

Comment:
@alcoat  - Apologies I didn't see that you had updated things before.  I've run the regression tests and was just doing a final check when I saw this addition.  I don't think EXTCDE is needed in this part of the code.  Am I missing something?